### ISSUE TITLE
[1.20.1][Forge/NeoForge] Weird crash with Balm 7.2.1

### ISSUE THREAD

**Issue by VOLKOUL** [ModdingForBlockheads/Forbic#74]
### Minecraft Version

1.20.1 (LTS)

### Mod Loader

NeoForge

### Mod Loader Version

NeoForge 47.1.100

### Mod Version

Balm 7.2.1

### Balm Version

Balm 7.2.1

### Describe the Issue

I was troubleshooting a modpack and it ran fine with Balm installed. Then, as I emptied the mods folder and started adding mods to find the causes of various unrelated issues, Balm suddenly decided to crash my game. (there are no mods present that depend on Balm btw)

### Logs

Crash Report: https://mclo.gs/cZ7Ig4I Log: https://mclo.gs/ZG5UBAM

### Do you use any performance-enhancing mods (e.g. OptiFine) or custom server distributions (e.g. SpongeForge)?

Yes, Embeddium. Every other mod loaded is in the Log

**Comment by VOLKOUL** [comment]
Crash occurs on startup btw

**Comment by BlayTheNinth** [comment]
I don't see a relation to Balm - the crash is coming from a ShetiPhianCore mixin failing to inject.

**Comment by VOLKOUL** [comment]
~~Hmm. So I took a wild guess. Previously I didn't have any mods that depend on either of these two installed. I assumed that ShetiPhianCore and Balm have beef and need at least one mod that depends on them to be present in order to work when both of them are installed. I added the Waystones mod and the Platforms mod (needs ShetiPhianCore) and it surprisingly worked! That is really weird. Could it be that ShetiPhianCore has some kind of incompatibility with Balm then?~~
